### DISCUSSION TITLE
in minified CSS why not combine and merge classes to be less size of code?

### DISCUSSION THREAD

**Discussion by webwamp** [twbs/bootstrap#34409] [General]
why in minified CSS version, you not combine / merge classes to have less size of code?
for example **Padding** and **Margin** and ... here this example:

`.py-0{padding-top:0!important;padding-bottom:0!important}`
`.pt-0{padding-top:0!important}` and `.pb-0{padding-bottom:0!important}`

so you can easily do this:
`.py-0,.pt-0{padding-top:0!important}` and `.py-0,.pb-0{padding-bottom:0!important}`

in this combine you have very minified and less size codes, so why you not do that? anybody can explain to me?

**Comment by XhmikosR** [comment]
We have experimented with such optimizations and the gains aren't too big to justify the potential breakage.
